rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -768,6 +768,12 @@ rm -f %{_builddir}/file.list.%{name}
 %config(noreplace) /etc/ld.so.conf.d/rudder.conf
 %endif
 
+%config(noreplace) /opt/rudder/etc/uuid.hive
+%config(noreplace) /etc/cron.d/rudder-agent
+%config(noreplace) /etc/init.d/rudder-agent
+%config(noreplace) /etc/default/rudder-agent
+%config(noreplace) /etc/profile.d/rudder-agent.sh
```

Comment:
Shouldn't we set /etc/default/rudder-agent as config(noreplace), as it is only there to let the user override variables in the init script?